tests/test_resolve_service.py

```
@@ -0,0 +1,370 @@
+"""Tests for the resolve service module."""
+
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)